atom/renderer/atom_renderer_client.cc

```diff
@@ -151,8 +151,7 @@ bool AtomRendererClient::ShouldFork(blink::WebLocalFrame* frame,
   // FIXME We only support GET here because http method will be ignored when
   // the OpenURLFromTab is triggered, which means form posting would not work,
   // we should solve this by patching Chromium in future.
-  *send_referrer = true;
-  return http_method == "GET";
+  return false;
```

Comment:
To add a note here, this PR is blocked because we have to return `true` here to make sure the renderer process is restarted for every navigation, however that causes crash for the changes in this PR.

I'm closing this PR since we don't really have a solution here.